### PR TITLE
Add 'update' function to merge two nodegraphs of the same size and ksize

### PR DESCRIPTION
**Pull request by ctb**
This is to support functionality necessary to implement Sequence Bloom Trees, (Solomon & Kingsford; https://github.com/Kingsford-Group/bloomtree).

### CODE REVIEW

**Comment by ctb** [comment]
- [ ] Is it mergeable?
- [ ] Did it pass the tests?
- [ ] If it introduces new functionality in scripts/ is it tested?
  Check for code coverage with `make clean diff-cover`
- [ ] Is it well formatted? Look at `make pep8`, `make diff_pylint_report`,
  `make cppcheck`, and `make doc` output. Use `make format` and manual
  fixing as needed.
- [ ] Did it change the command-line interface? Only additions are allowed
  without a major version increment. Changing file formats also requires a
  major version number increment.
- [ ] Is it documented in the ChangeLog?
  http://en.wikipedia.org/wiki/Changelog#Format
- [ ] Was a spellchecker run on the source code and documentation after
  changes were made?
- [ ] Is the Copyright year up to date?

**Comment by ctb** [comment]
Ready for review and merge @camillescott @luizirber

File: khmer/_khmermodule.cc [hunk 1]
```diff
@@ -3078,8 +3078,37 @@ hashbits_count_overlap(khmer_KHashbits_Object * me, PyObject * args)
...
+
+    try {
+        hashbits->update_from(*other);
+    } catch (khmer_exception &e) {
+        PyErr_SetString(PyExc_ValueError, e.what());
+        return NULL;
+    }
+
+    Py_INCREF(Py_None);
+    return Py_None;
```
Comment (camillescott) [inline khmer/_khmermodule.cc]: should use Py_RETURN_NONE instead of reference counting yourself

**Comment by ctb** [comment]
@camillescott fixed!

**Comment by camillescott** [comment]
LGTM; merging

**Comment by ctb** [comment]
thanks!

**Pull request merged**